docs/src/main/resources/Managing.adoc

```diff
@@ -1168,6 +1168,280 @@ TLS_RSA_WITH_AES_128_CBC_SHA
+====
+
+In <{branding}_HOME>/etc/system.properties, modify the keystore and truststore paths and passwords as appropriate.
+
+.system.properties
+[source,bash]
+----
+#START DDF SETTINGS
+# Set the keystore and truststore Java properties
+javax.net.ssl.keyStore=etc/keystores/serverKeystore.jks
+javax.net.ssl.keyStorePassword=<NewPassword>
+javax.net.ssl.trustStore=etc/keystores/serverTruststore.jks
+javax.net.ssl.trustStorePassword=<NewPassword>
+javax.net.ssl.keyStoreType=jks
+----
+
+In <{branding}_HOME>/etc/ws-security/issuer, modify encryption.properties
+
+.encryption.properties
+[source,bash]
+----
+#Property descriptions at http://ws.apache.org/wss4j/config.html
+org.apache.ws.security.crypto.provider=org.apache.ws.security.components.crypto.Merlin
+org.apache.ws.security.crypto.merlin.keystore.type=jks
+org.apache.ws.security.crypto.merlin.keystore.password=<NewPassword>
+org.apache.ws.security.crypto.merlin.keystore.alias=<Alias>
+org.apache.ws.security.crypto.merlin.keystore.file=etc/keystores/serverKeystore.jks
+----
+
+In <{branding}_HOME>/etc/ws-security/issuer, modify signature.properties
+
+.signature.properties
+[source,bash]
+----
+#Property descriptions at http://ws.apache.org/wss4j/config.html
+org.apache.ws.security.crypto.provider=org.apache.ws.security.components.crypto.Merlin
+org.apache.ws.security.crypto.merlin.keystore.type=jks
+org.apache.ws.security.crypto.merlin.keystore.password=<NewPassword>
+org.apache.ws.security.crypto.merlin.keystore.private.password=<NewPassword>
+org.apache.ws.security.crypto.merlin.keystore.alias=<Alias>
+org.apache.ws.security.crypto.merlin.keystore.file=etc/keystores/serverKeystore.jks
+----
+
+In <{branding}_HOME>/etc/ws-security/server, modify encryption.properties
+
+.encryption.properties
+[source,bash]
+----
+#Property descriptions at http://ws.apache.org/wss4j/config.html
+org.apache.ws.security.crypto.provider=org.apache.ws.security.components.crypto.Merlin
+org.apache.ws.security.crypto.merlin.keystore.type=jks
+org.apache.ws.security.crypto.merlin.keystore.password=<NewPassword>
+org.apache.ws.security.crypto.merlin.keystore.alias=<Alias>
+org.apache.ws.security.crypto.merlin.keystore.file=etc/keystores/serverKeystore.jks
+#org.apache.ws.security.crypto.merlin.x509crl.file=etc/certs/demoCA/crl/crl.pem
+----
+
+In <{branding}_HOME>/etc/ws-security/server, modify signature.properties
+
+.signature.properties
+[source,bash]
+----
+#Property descriptions at http://ws.apache.org/wss4j/config.html
+org.apache.ws.security.crypto.provider=org.apache.ws.security.components.crypto.Merlin
+org.apache.ws.security.crypto.merlin.keystore.type=jks
+org.apache.ws.security.crypto.merlin.keystore.password=<NewPassword>
+org.apache.ws.security.crypto.merlin.keystore.alias=<Alias>
+org.apache.ws.security.crypto.merlin.keystore.file=etc/keystores/serverKeystore.jks
+----
+
+==== Creating New Certificates
+===== Create private key and certificate signing request
+
+.Create private key
```

Comment:
Broke the creating new certificate section into a creation and importing section. Added notes and links on where to skip to based on what you are trying to do.